openpathsampling/analysis/tis_analysis.py

```diff
@@ -353,6 +343,9 @@ def total_crossing_probability(self, storage=None, method="wham", force=False):
         force : bool (False)
             if true, cached results are overwritten
         """
+
+        tcp = dict()
```

Comment:
Don't do this. If WHAM fails to return a correct value, we want to give an error, not to return an empty LookupFunction. In general, I think it is better to let python raise an error if something goes wrong, instead of giving a number which will then be wrong (and possibly published by an inattentive user!)